byob/README.md

```diff
@@ -0,0 +1,72 @@
+# BYOB CLI
+These instructions help you solve some of the most common problems that may arise during the installation and use of BYOB CLI. However, if something does not work despite everything, before opening an issue try to ask on the [Discord support server](https://discord.gg/8FsSrw7)!
+
+## How to install
+To install BYOB CLI run the _setup.py_ file through your Python installation (Python3 is recommended): `python3 setup.py`. Done, it's that simple! However, there may be problems with your OS environment, and some cases are covered in the following sections.
+
+### Linux
+Problems that can occur when installing BYOB in a Unix environment are, for example, missing packages or tools required to build these packages. Of course, you need to install Python3, to run the code, and Pip, to install the packages. Some of these require _CMake_ to be installed, along with other system build tools. The following bash commands do essentially the same thing as the _setup.py_ file but should cover all possible failure scenarios. Suppose we are root users.
+
+```bash
+$ git clone https://github.com/malwaredllc/byob.git
+$ cd byob/byob/
+# First, Python3
+$ apt install python3.6 # 3.7, 3.8, 3.9 should also work
```

Comment:
I used "should" because I wasn't sure about the compatibility, but from what I have personally been able to see, everything works fine on Python 3.9(.1). I obviously haven't tried every module and every line of code, but I had no problems starting the server, generating the client, connecting, performing some operations via the shell...